plugins/portlet-mocks/src/test/java/org/apache/struts2/StrutsSpringPortletMockObjectsTest.java

```diff
@@ -222,7 +222,8 @@ public void testMockClientDataRequest() {
         assertEquals("Content-type not as expected ?", TEST_CONTENT_TYPE, mockClientDataRequest.getContentType());
         mockClientDataRequest.setMethod(TEST_METHOD);
         assertEquals("Method not as expected ?", TEST_METHOD, mockClientDataRequest.getMethod());
-        try ( InputStream inputStream = mockClientDataRequest.getPortletInputStream() ) {
+	InputStream inputStream = mockClientDataRequest.getPortletInputStream();
```

Comment:
This change in the PR should probably be reverted.  The previous try-with-resources call is safer, and restoring that code should eliminate the `IOException` issue currently being reported during the compilation.

Reply:
Thanks for the suggestion, the PR has now been updated!